src/Resources/views/http_message.html.twig

```diff
@@ -0,0 +1,29 @@
+{% set hasReachedBody = false %}
+{% set content = '' %}
+{% set data = data|split("\n")|slice(1) %}
+{% set xdebugTokenLink = data|filter(v => 'x-debug-token-link:' in v|lower)|first|split(':')|slice(1)|join(':')|trim %}
```

Comment:
could you instead split with a limit of 1, to not break up the url and then have to re-join it? would be more elegant imho. `data|filter(v => 'x-debug-token-link:' in v|lower)|first|split(':', 2)|slice(1)|trim`

Reply:
I came up with even better simplification. WDYT?